### DISCUSSION TITLE
Horizontal Chart

### DISCUSSION THREAD

**Discussion by damiike** [ColorlibHQ/AdminLTE#4302] [Q&A]
Hi All,

I really like this theme and am looking at buying Pro.

I have one question. I am looking to use the theme, but I also need a way to create a horizontal multi calendar similar to this:

https://res.cloudinary.com/softwarepundit/image/upload/c_limit,dpr_1.0,f_auto,h_1600,q_auto,w_1600/v1/software/guesty_centralized_calendar

or 

https://files.readme.io/2fae3f8-calendar.png

Is that possible with any of the existing elements available in AdminLTE Pro?

If not, can anyone point me in the right direction for what can be used with AdminLTE Pro to produce it?

Thank you so much!

**Comment by lenamtl** [comment]
Hi,

I would try to do this using Datatables that will be a big challenge but this can be powerful once it work
Here is some example to start
https://datatables.net/forums/discussion/comment/196640/#Comment_196640
I would use in combinaison of responsive

You can check here this is non conventionnal use of Datatables so maybe someone post something for calendar.
https://datatables.net/forums/discussion/230/how-are-you-using-datatables#latest

Or you can use Fullcalendar
https://fullcalendar.io/demos

**Comment by lenamtl** [comment]
Hi, 
you can also use d3.js 
https://blog.risingstack.com/tutorial-d3-js-calendar-heatmap/
https://blog.logrocket.com/using-d3-to-create-a-calendar-app/

But anyway AdminLTE is juste a template ...
You can add any external script to achieve what you need but this will be custom code for sure...
